startups/serializers.py

```diff
@@ -28,6 +29,7 @@ class Meta:
         read_only_fields = ('registration_date', 'owner')
 
     def create(self, validated_data):
+        print(validated_data)
         user = self.context['request'].user
```

Comment:
I do not think this `print `is necessary.

Reply:
done